packages/ui-extensions/src/surfaces/checkout/components/Section/Section.doc.ts

```diff
@@ -0,0 +1,47 @@
+import type {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+import sharedContent from '../../../../docs/shared/components/Section';
+
+const data: ReferenceEntityTemplateSchema = {
+  ...sharedContent,
+  thumbnail: '',
+  isVisualComponent: true,
+  subSections: [
+    {
+      title: 'Useful for',
+      type: 'Generic',
+      anchorLink: 'useful-for',
+      sectionContent: `- Organizing your page in a logical structure based on nesting levels.
+- Creating consistency across pages.`,
+    },
+    {
+      title: 'Considerations',
+      type: 'Generic',
+      anchorLink: 'considerations',
+      sectionContent: `- When adding headings inside sections they automatically use a specific style, which helps keep the content organized and clear.
+- Built-in spacing is added between nested sections, as well as between heading and content.
+- **Level 1:** Display as responsive cards with background color, rounded corners, and shadow effects. Includes outer padding that can be removed when content like \`s-table\` needs to expand to the full width of the section.
+- **Nested sections:** Don't have any background color or effects by default.`,
```

Comment:
```suggestion
      sectionContent: `When adding headings inside sections they automatically use a specific style, which helps keep the content organized and clear.`,
```

We should remove these 3 bullets. They reference designs only done in Admin.

Reply:
No problem at all. 

I've removed the bottom three points, you are right that the nesting level context is mentioned in the shared description: 

> Sections have contextual styling that automatically adapts based on nesting depth.